CONTRIBUTING.rst

```diff
@@ -34,7 +34,7 @@ Here’s what the pull request process looks like in brief:
 
 To learn more about basic requirements and standards of code contribution, please familiarize yourself with our comprehensive `documentation <https://docs.django-cms.org/en/latest/contributing/code.html#>`_.
 
-django CMS commits follow the `Conventionnal Commits Guideline <http://conventionnalcommits.org/>`_ , please try to follow the Guidelines in your commit messages to ease our review&merge process.
+django CMS commits follow the `Conventionnal Commits Guideline <https://conventionalcommits.org/>`, please try to follow the Guidelines in your commit messages to ease our review&merge process.
```

Comment:
```suggestion
django CMS commits follow the `Conventional Commits Guideline <https://conventionalcommits.org/>`_, please try to follow the Guidelines in your commit messages to ease our review & merge process.
```

The underscore is important here; https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
Ahh thank you for this, not too familiar with .rst files.